lib/WIFI/WIFI.h

```diff
@@ -92,7 +92,7 @@ class WIFI
     bool connectStation();
 
     /** Timeout for WiFi connection. */
-    const unsigned long WIFI_TIMEOUT_MS = 30000;
+    const unsigned long WIFI_TIMEOUT_MS;
 
     /** True if system is connected in STA mode to a Network. */
     bool isStaAvailable = false;
```

Comment:
So this one for being a variable should be initialized on the initialization List?

Reply:
If its a member variable (not a constant) yes, but in this case the member variable name is invalid.
But my assumption according to "const" and name was, it shall be really a constant. In this case and if you add "static", you can't initialize it in the initialization list. :-)

Reply:
Therefore, Line 98 should only be `bool m_isStaAvailable;`  so it is according to coding style and its initialized in the initialization list as it is not a constant?

Reply:
Yes, exactly.

Reply:
Don't forget to sleep.